control/grpc.py

```diff
@@ -48,6 +48,8 @@
 CNTLID_RANGE_SIZE = 2040
 DEFAULT_MODEL_NUMBER = "Ceph bdev Controller"
 
+MAX_POSSIBLE_ANA_GRPS = 100
```

Comment:
@leonidc great work!

Please remove `MAX_POSSIBLE_ANA_GRPS` arbitrary constant.

Reply:
what do you suggest? I use this to initialize the dictionary for simplify the code . is number 100 too small for total number of ana groups?

Reply:
@leonidc maybe read the limit from the conf file?

Reply:
max number ANA groups = max number GWs in the pool. do we have this limitation in config? if no I can add it to config

Reply:
we can add a config parameter - max_gws_in_grp. That should be defaulted to 16 upstream. You can use that param.

Reply:
yes. good contstant.  will add